attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -138,4 +118,44 @@ value_fwi_shrimp =
   value_of_shrimp_in_fish
 
 value_fwi_so_far = value_fwi_fish + value_fwi_shrimp
-value_fwi_so_far
+
+fwi_estimates = {
+  name: "fwi",
+	value: value_fwi_so_far,
+}
+
+/*
+## Open Wing Alliance
+
+Sources: 
+- <https://animalcharityevaluators.org/charity-review/the-humane-league/#comprehensive-review>
+- <https://thehumaneleague.org/article/owa-year-in-review-2022>
+
+What am I estimating: Impact of corporate commitments that OWA made happen in 2022.
+
+Notes on the estimation: The number of corporate campaigns is from OWA's own reporting. The number of chickens affected and the improvement per chicken is pretty much pulled out of my ass.
```

Comment:
> The number of chickens affected and the improvement per chicken is pretty much **pulled out of my ass.**

Can we have some other terminology here? Maybe use "levels of how speculative something is" or similar?

Lots of stuff here will be very highly speculative.

Reply:
Yes